### ISSUE TITLE
Arcane mage priorities missing from latest update

### ISSUE THREAD

**Issue by BearminghamUK** [Hekili/hekili#2142]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.

### WoW Version

Retail (Dragonflight)

### Describe the Issue

Fire and frost mages are loading - arcane mages missing from today's update. Yesterday was working. Thank you

### How to Reproduce

1. load game as arcane mage
2. Arcane priorities missing from list

### Player Information (Link)

```Text
build: v10.0.2-2.0.5a
level: 70 (70)
class: MAGE
spec: arcane

talents: none

pvptalents: none

covenant: none

conduits: none

soulbinds: [pelagos]

sets: none

gear: belt_of_living_earth = 1
    circle_of_ascended_frost = 1
    cloak_of_lost_devotion = 1
    cobalt_watchers_cowl = 1
    cobalt_watchers_cuffs = 1
    cobalt_watchers_handwraps = 1
    cobalt_watchers_leggings = 1
    cobalt_watchers_sandals = 1
    cobalt_watchers_shoulderpads = 1
    cobalt_watchers_vestment = 1
    grals_discarded_tooth = 1
    guild_tabard = 1
    heart_of_azmerloth = 1
    maruuk_battle_standard = 1
    scarlet_filigreed_shirt = 1
    unstable_arcane_loop = 1
    windsealed_mana_capsule = 1

legendaries: none

itemIDs: 167188, 191983, 191984, 191986, 191987, 191988, 191990, 192015, 192033, 192797, 193629, 193633, 193731, 195210, 200736, 200880, 5976

settings: none

toggles: cooldowns = false 
    custom1 = false 
    custom2 = false 
    defensives = false 
    essences = true 
    interrupts = false 
    mode = automatic 
    potions = false 

keybinds: 

warnings: [#1] Unknown buff in [Frost Mage:movement:2]: ice_floes
[string "@Interface/AddOns/Hekili/State.lua"]:3821: in function <Interface/AddOns/Hekili/State.lua:3812>
[string "return buff.ice_floes.down"]:1: in main chunk
[string "=[C]"]: in function `pcall'
[string "@Interface/AddOns/Hekili/Scripts.lua"]:921: in function `EmulateSyntax'
[string "@Interface/AddOns/Hekili/Scripts.lua"]:1196: in function <Interface/AddOns/Hekili/Scripts.lua:1187>
[string "@Interface/AddOns/Hekili/Scripts.lua"]:1508: in function `LoadScripts'
[string "@Interface/AddOns/Hekili/Scripts.lua"]:1597: in function `LoadScripts'
[string "@Interface/AddOns/Hekili/Core.lua"]:1480: in function <Interface/AddOns/Hekili/Core.lua:1478>

[#2] Unable to compile 'travel_time' - Interface/AddOns/Hekili/State.lua:2175: attempt to index local 'ability' (a nil value) (pcall-n)
From: variable.time_to_combustion <= 0 | buff.combustion.remains > travel_time | ( cooldown.meteor.duration < variable.time_to_combustion & ! talent.rune_of_power.enabled ) | talent.rune_of_power.enabled & buff.rune_of_power.up & variable.time_to_combustion > action.meteor.cooldown | fight_remains < variable.time_to_combustion
[#3] Unable to compile 'action.meteor.cooldown' - [string "return action.meteor.cooldown"]:1: attempt to index field 'meteor' (a nil value) (pcall-n)
From: variable.time_to_combustion <= 0 | buff.combustion.remains > travel_time | ( cooldown.meteor.duration < variable.time_to_combustion & ! talent.rune_of_power.enabled ) | talent.rune_of_power.enabled & buff.rune_of_power.up & variable.time_to_combustion > action.meteor.cooldown | fight_remains < variable.time_to_combustion
[#4] Unable to compile 'travel_time' - Interface/AddOns/Hekili/State.lua:2175: attempt to index local 'ability' (a nil value) (pcall-n)
From: variable.time_to_combustion<=0|buff.combustion.remains>travel_time|(cooldown.meteor.duration<variable.time_to_combustion&!talent.rune_of_power.enabled)|talent.rune_of_power.enabled&buff.rune_of_power.up&variable.time_to_combustion>action.meteor.cooldown|fight_remains<variable.time_to_combustion
[#5] Unable to compile 'action.meteor.cooldown' - [string "return action.meteor.cooldown"]:1: attempt to index field 'meteor' (a nil value) (pcall-n)
From: variable.time_to_combustion<=0|buff.combustion.remains>travel_time|(cooldown.meteor.duration<variable.time_to_combustion&!talent.rune_of_power.enabled)|talent.rune_of_power.enabled&buff.rune_of_power.up&variable.time_to_combustion>action.meteor.cooldown|fight_remains<variable.time_to_combustion
[#6] Error in Fire:active_talents:2 conditions:  2175: attempt to index local 'ability' (a nil value)
```

### Error Messages (Link)

```Text
no error messages
```

**Comment by lakeriod** [comment]
Dude, just read last changelog

**Comment by Hekili** [comment]
Arcane Mage is temporarily disabled to remove its debug messages while I work to enable its latest "phased" priority lists.  I'm hoping to post an updated release today.

**Comment by BearminghamUK** [comment]
Perfect. Thank you.

Sent from my iPhone

On 14 Dec 2022, at 14:14, Hekili ***@***.***> wrote:

﻿

Closed #2142<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FHekili%2Fhekili%2Fissues%2F2142&data=05%7C01%7C%7Cade2d2bc7d3c42556ba408dadddd7912%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638066240539480857%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=N9kdffioH1EBlHliVcJDNm4jULlEiCFPzL8wUAc%2BMfg%3D&reserved=0> as completed.
